### DISCUSSION TITLE
Implement EVENT_REGISTER_FORM_ACTIONS on Orders

### DISCUSSION THREAD

**Discussion by nixondesigndev** [craftcms/commerce#2469] [Ideas]
In Craft 3.6.10 `craft\events\FormActionsEvent` and `craft\web\twig\variables\Cp::EVENT_REGISTER_FORM_ACTIONS` were added in response to https://github.com/craftcms/commerce/pull/2035.

Could we have something similar for the settings dropdown on order pages along with the existing delete options? This will be useful as we have clients where they often need to perform extra actions on an order.

**Comment by nfourtythree** [comment]
Hi @NixonGit 

I am just looking to clarify what you require from this. Are you talking about the orders index page?

If so this is already possible with `Element::EVENT_REGISTER_ACTIONS` see: https://craftcms.com/docs/3.x/extend/element-action-types.html

Let me know if this is the solution you are looking for, thanks.

**Comment by nixondesigndev** [comment]
@nfourtythree We were looking at trying to add extra items under the "cog" menu on an individual order.

![edit-order](https://user-images.githubusercontent.com/17004710/122422755-5050ba80-cf85-11eb-8dbf-f3c37fceb136.png)

**Comment by nfourtythree** [comment]
Hi @NixonGit 

Thank you for clarifying!
